### PR TITLE
GEOMESA-792 Regress Kryo to 2.21 from 3.0

### PR DESCRIPTION
**Pull request by cne1x**
Removed our internal (in|ex)clusion for relocated JARs.

Signed-off-by: Chris Eichelberger <cne1x@ccri.com>

### CODE REVIEW

File: geomesa-feature/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (elahrvivaz) [inline geomesa-feature/dependency-reduced-pom.xml]: we don't want this file anymore, right?

File: geomesa-feature/pom.xml [hunk 2]
```diff
@@ -36,9 +36,9 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.esotericsoftware</groupId>
-            <artifactId>kryo-shaded</artifactId>
-            <version>3.0.0</version>
+            <groupId>com.esotericsoftware.kryo</groupId>
+            <artifactId>kryo</artifactId>
+            <version>2.21</version>
```
Comment (elahrvivaz) [inline geomesa-feature/pom.xml]: did you make sure that kryo 3 isn't coming in from somewhere else? i thought that there was a line of code that would have to be changed for kryo 2...

File: geomesa-feature/dependency-reduced-pom.xml (hunk 1 above)
Comment (cne1x) [inline geomesa-feature/dependency-reduced-pom.xml]: Removed this file.

File: geomesa-feature/pom.xml (hunk 2 above)
Comment (cne1x) [inline geomesa-feature/pom.xml]: Good question.  I've rebuilt (with tests), and not seen any complaints.  As a check, I ran the following:

```
11:32 $ mvn dependency:tree | grep -i kryo
[INFO] +- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:provided
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:provided
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
```

Looks reasonable to me, but if I'm overlooking something, please just holler.

**Comment by jnh5y** [comment]
Merged as 9ddcf43bf697175866ff2b8e746463ae00e7a2a1 on 1.5.

**Pull request closed without merging**